api/server/router/network/filter.go

```diff
@@ -29,6 +29,29 @@ func filterNetworkByType(nws []types.NetworkResource, netType string) ([]types.N
 	return retNws, nil
 }
 
+func filterNetworkByUse(nws []types.NetworkResource, danglingOnly bool) ([]types.NetworkResource, error) {
+	retNws := []types.NetworkResource{}
+
+	switch danglingOnly {
```

Comment:
@vdemeester I'm ok with changing to that, but that wouldn't be consistent with the existing code, that's all.